content/docs/tools/teddyCloud/tonies-custom-json.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "tonies-custom-json config"
+description: "Info about the tonies-custom-json file"
+bookCollapseSection: true
+---
+# tonies.custom.json
+Toniecloud uses the tonies-custom-json file to read the metadata of custom tags in the same manner it's done for the official boxine tonies in the tonies-json file. The structure is the same, but it's not overwritten as the tonies-json file through regularly updates. So you can use the tonies-custom-json to save metadata of your own custom tags.
+
+Initially the tonies-custom-json file looks like the following:
+
+```
+[]
+```
+
+In the gui, a custom tag looks initially like:
+
+_old gui_
+
+![Old GUI](/img/tonies-custom-json_empty_oldgui.png)
+
+
+_new /web gui_
+
+![New GUI](/img/tonies-custom-json_empty_newgui.png)
+
+
+
+With one custom tag the tonies-custom-json file looks like this:
+
+```
+[{"no": "0", "model": "123456", "audio_id": ["369519776"], "hash": ["af9e61a9c1b12138fb060908d595742334b04515"], "title": "Custom Tonie Example Title", "series": "Custom Tonies", "episodes": "This is my custom tonie", "tracks": ["Title 1", "Title 2", "Title 3", "Title 4", "Title 5", "Title 6", "Title 7", "Title 8", "Title 9", "Title 10"], "release": "0", "language": "de-de", "category": "custom", "pic": "https://upload.wikimedia.org/wikipedia/en/6/6b/Hello_Web_Series_%28Wordmark%29_Logo.png"}]
```

Comment:
It may be easier for a user to understand, if the example is broken into multiple lines (formatted / beautified json)

Reply:
i added a link to the section with more details...